sdl3-config.in

```diff
@@ -46,7 +46,7 @@ while test $# -gt 0; do
       echo @SDL_VERSION@
       ;;
     --cflags)
-      echo -I@includedir@/SDL3 @SDL_CFLAGS@
+      echo -I@includedir@ -I@includedir@/SDL3 @SDL_CFLAGS@
```

Comment:
Is it okay if I remove all `-I@includedir@/SDL3` from all sdl3 config scripts (pkg-config, sdl3-config, cmake)?

Reply:
```suggestion
      echo -I@includedir@ @SDL_CFLAGS@
```

Reply:
> Is it okay if I remove all `-I@includedir@/SDL3` from all sdl3 config scripts (pkg-config, sdl3-config, cmake)? Or is it too harsh?

I don't think that it'd be harsh.

Reply:
Yes, that's fine, although I'm about to remove sdl3-config, so you can skip that one.